src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -552,9 +552,13 @@ private static async Task<string> GetComponentBuildDropLogAsync(Build newestBuil
 
         internal static async Task<(List<GitCommit> changes, string diffLink)> GetChangesBetweenBuildsAsync(Build fromBuild, Build tobuild, CancellationToken cancellationToken)
         {
-            if (tobuild.Repository.Type == "GitHub")
+            var type = tobuild.Repository.Type;
+            if (type == "GitHub" || Options.ComponentBuildProjectName == "internal")
```

Comment:
Also @JoeRobich,
Is this enough to distinguish dnceng release?

Reply:
Please review https://github.com/dotnet/roslyn-tools/pull/982 which adds commit log generation for AzDO repos.